readme.md

```diff
@@ -111,53 +111,57 @@ $ npm -v
 
 * **Question:** What gets logged? Why?
 
-* **Your Answer:**
+* **Your Answer:** 'Jian' aka '<your-name>' gets logged because it console logs the exported profile string.
 
 * **Question:** What is `module.exports` and what is its _type_ in JavaScript? What is `require` and what is its _type_ in JavaScript?
 
-* **Your Answer:**
+* **Your Answer:** module.exports and require are a functions. 
+
+* The module.exports or exports is a special object which is included in every JS file in the Node.js application by default. module is a variable that represents current module and exports is an object that will be exposed as a module. So, whatever you assign to module.exports or exports, will be exposed as a module. 
+
+* require() is not part of the standard JavaScript API. But in Node.js, it's a built-in function with a special purpose: to load modules. In Node.js, each module has its own scope. A module cannot directly access things defined in another module unless it chooses to expose them. To expose things from a module, they must be assigned to exports or module.exports. For a module to access another module's exports or module.exports, it must use require().
```

Comment:
Please answer [in your own words](https://stackoverflow.com/questions/9901082/what-is-this-javascript-require#9901097).